### ISSUE TITLE
Inaccurate Documentation for Setting up Perf Toolkit

### ISSUE THREAD

**Issue by devoredevelops** [Microsoft/CRM-Performance-Toolkit#27]
In the Word document "Microsoft Dynamics CRM 2015 Performance Toolkit" step 2 under the section "Microsoft CRM 2015 Performance Toolkit installation" states, "The version of CRM SDK to install should match the version of the deployed CRM that you will run the load test against." Following this advice for CRM 2013 (and using the SDK for CRM 2013), I could not get the performance toolkit to build. The code referenced too many classes that don't exist in the assembles from the CRM 2013 SDK. When I used the CRM 2015 SDK, the project builds successfully.

I believe the Word document should be updated to reflect this. Thanks!

**Comment by jeffreyzhzhang** [comment]
Hi,

Could you find the Solution Configuration that matches the deployment version to be tested?  You can find all the supported solution configurations from Visual Studio by clicking on Solution -> Configuration Manager. I believe the toolkit supports  CRM version of 2013SP1 and onwards if I remember correctly. Please note the toolkit on GitHub was not actively maintained. If you need to run load test against your CRM deployment, you’d better work with someone from Microsoft field engineering team or relative teams.

Thanks,
Jeffrey

**Comment by rafilks** [comment]
same problem

**Comment by devoredevelops** [comment]
@jeffreyzhzhang Yes I was able to find that solution configuration. Fortunately I'm able to test against a CRM2016 server using the CRM2015SP1 solution configuration. Most of the Create and Update web/unit tests in the solution don't work due to values not getting populated in command queries executed by EntityManager, but that's another story. Thank you for your help!

**Comment by jules09235** [comment]
@jeffreyzhzhang @sqyttles Has anyone seen this error? We are running CRM 2015 and I have SDK 2015 too. 

Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe C:\crmtoolkit\CRM_Perf_Toolkit\CRM_Perf_Toolkit.sln /P:Configuration=CRM2015 /noconsolelogger failed
-1
